site/content/docs/design/node-image.md

```diff
@@ -16,7 +16,12 @@ nested containers, systemd, and Kubernetes components.
 This image is built on top of the ["base" image][base image].
 
 Logic for building ["node" image][node image] can be found in [`pkg/build`][build package],
-and it can be built with `kind build node-image` respectively.
+and it can be built with `kind build node-image` respectively.  Building this
+image via the `kind` command requires both the `kind` and the `kubernetes`
+projects' source to be downloaded into the `GOPATH`.  The kind source can be
+installed via the command `GO111MODULE="on" go get sigs.k8s.io/kind@v0.8.0`.  To
```

Comment:
let's not duplicate installation instructions, it's hard to keep these all in sync. the main guide has install information, install information doesn't belong in the design doc.